src/gui/static/src/app/components/pages/transaction-list/transaction-list.component.ts

```diff
@@ -38,4 +39,14 @@ export class TransactionListComponent implements OnInit, OnDestroy {
     config.data = transaction;
     this.dialog.open(TransactionDetailComponent, config).afterClosed().subscribe();
   }
+
+  showQrCode($event: any, address: string[]) {
```

Comment:
Are you sure that you can just always show the first address in this case?

Reply:
There can be multiple addresses in a transaction, so I decided to add the ability to show a QR code only when there is exactly one.

I could transform multiple addresses into list (instead of a paragraph) and have the qrcode icon next to each other (that might take a lot of space and look weird)